lib/std/os/uefi.zig

```diff
@@ -43,7 +43,7 @@ pub const Guid = extern struct {
 };
 
 /// An EFI Handle represents a collection of related interfaces.
-pub const Handle = *@OpaqueType();
+pub const Handle = ?*@OpaqueType();
```

Comment:
Wouldn't it make more sense to leave this as non-optional, and then use `?Handle` in the places where it can be `null`, and not otherwise? I'll find some places and comment where it looks like it should be non-optional.

Reply:
I can't seem to find the exact reason why I did this. I should have put this in a separate commit with some explanation 🤔 I'm going to remove this change later today and check every use of Handle in this PR.